src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,534 @@
+          recordType: null,
+          headers: {
+            approvalRequired: [
+              {
+                text: "Date last modification",
+                value: "updatedAt"
+              },
+              {
+                text: "Curator",
+                value: "curator"
+              },
+              {
+                text: "Record name (id)",
+                value: "recordName"
+              },
+              {
+                text: "Last editor",
+                value: "lastEditor"
+              }
+            ],
+            maintenanceRequests: [
+              {
+                text: "Date",
+                value: "createdAt"
+              },
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "User login (id)",
+                value: "userNameID"
+              }
+            ],
+            recordsCreatedCuratorsLastWeek: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Creator",
+                value: "creator"
+              }
+            ],
+            recordsInCuration: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Record maintainer(s)",
+                value: "recordMaintainers"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              }
+            ],
+            recordsWithoutDois: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Date last edit",
+                value: "updatedAt"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              },
+              {
+                text: "Last edited by",
+                value: "lastEditor"
+              }
+            ],
+            hiddenRecords: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              }
+            ]
+          },
+          search1: "",
+          search2: "",
+          search3: "",
+          search5: "",
+          search6: "",
+          search7: ""
```

Comment:
Use an object containing  all the search fields rather than a separate field per search. Use semantically meaningful names, e.g. names that refer to the field you are serching
```suggestion
          searches: {
              approvalRequired: '',
              maintenanceRequests: '',
              recordsCreatedCuratorsLastWeek: '',
              recordsInCuration: '',
              recordsWithoutDois: '',
              hiddenRecords: ''           
          }
```